docs/course_materials/modules_packages.html

```diff
@@ -49,10 +49,23 @@ <h1><b>
                     <h3><b>The 'import' statement</b></h3>
                     <ul>
                         <li>Allow use of other python files and libraries</li>
-                        <li>Imports: import math</li>
-                        <li>Named imports: import math as m</li>
-                        <li>Specific imports: from math import pow</li>
-                        <li>Import all: from math import * (dangerous! used only in very specific cases)</li>
+
+                        <li>Imports: </li>
+                        <pre><code data-trim data-noescape class="hljs lang-python">
+                        import math
```

Comment:
I think it's better that all will be in 1 pre-code, and add notes with '#'

Reply:
disagree, because it's shouldn't be in the same file, it's different options/

Reply:
we can see the feedbacks after the class :)